recipes/lfzip-nlms/meta.yaml

```diff
@@ -0,0 +1,45 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+{% set name = "LFZip" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/shubhamchandak94/LFZip/archive/v{{ version }}.tar.gz
+  sha256: ef9b9b00f984daab9d65927232a5c0b2c7f21059fd7b26070aed02215f167501
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - make
+  host:
+    - python
+    - pip
```

Comment:
You don't need `python` and `pip` to build the software right?

Reply:
Not sure, I just need numpy during execution, so I thought I needed to put it here.

Reply:
If so, you need it only in `run`. You can keep `host` empty.

Reply:
Thanks, I have removed the host section and the tests still succeed.